asserts/snapasserts/validation_sets.go

```diff
@@ -85,8 +85,14 @@ func NewValidationSetKey(vs asserts.ValidationSet) ValidationSetKey {
 	return ValidationSetKey(strings.Join(vs.Ref().PrimaryKey, "/"))
```

Comment:
I missed this in the previous review, it's very unusual to pass assertions by value, this should take a *asserts.ValidationSet

Reply:
Fixed now